pkg/provider/pods.go

```diff
@@ -404,3 +405,71 @@ func (p *Pod) IsRunAsUserID(uid int64) bool {
 	}
 	return *p.Pod.Spec.SecurityContext.RunAsUser == uid
 }
+
+// Get the list of top owners of pods
```

Comment:
This would probably be a good candidate for unit testing.

Reply:
Will add the unit test in a separate PR